### DISCUSSION TITLE
Quartz.NET + MySqlConnector + Dependency Injection

### DISCUSSION THREAD

**Discussion by AaronBeier** [quartznet/quartznet#2613] [Q&A] [answered]
I'm using DI and have already added a MySQL Data Source using `Services.AddMySqlDataSource(...);`, how would i go about telling Quartz.NET to use said Data Source, instead of a new one?

**Comment by lahma** [comment, marked as answer]
Maybe this PR shows how to achieve this with MySQL too: https://github.com/quartznet/quartznet/pull/2438

**Reply by AaronBeier** [reply]
Works. Thanks! What helped me was this snippet: https://github.com/quartznet/quartznet/pull/2438/files#diff-5c8d836d7632ebba09b002188dc702d1baa0fe130f5fbd5a788e9e36be570f98R111-R120
